README.md

```diff
@@ -45,115 +89,169 @@ environment variable `EXTRA_PKGS`. See [Environment Variables](#Environment Vari
+Linux itself is not an `rpm`-based Linux distribution (the software installed
+on a Clear Linux system is not managed using `rpm`).
 
-Clear Linux is not an rpm-based Linux distribution and
-the rpm packages are not signed, so there is no way
-to ensure that downloaded packages are trustworthy.
+The `rpm` packages used to generate the rootfs are not signed, so there is no
+way to ensure that downloaded packages are trustworthy.
 
-If you are willing to use Clear Linux based images, we encourage you
-to use the Clear Containers images provided from its website
-https://download.clearlinux.org/current/.
+If you are willing to use Clear Linux based images, official Clear Containers
+rootfs images can be obtained from https://download.clearlinux.org/releases.
 
-## Build guest kernel ##
+## Build guest kernel
 
-Clear Containers uses the Linux kernel, you can build a
-kernel compatible with Clear Containers using the make
-`kernel` target. This will clone the [Clear Container Kernel]
-if the directory `workdir/linux` does not exist, then will build it.
-On success the new kernel will be located in
-workdir/vmlinux.container
+Clear Containers uses the [Linux* kernel](https://www.kernel.org).
+
+To build a kernel compatible with Clear Containers using the make `kernel` target. This
+will clone the [Clear Container Kernel] in the `workdir/linux` directory
+(which will be created if necessary). On success two new kernel images will be created:
+  - `workdir/vmlinuz.container` (compressed kernel image)
+  - `workdir/vmlinux.container` (uncompressed kernel image)
 
 
 ```
-#Pull and setup latest kernel for Clear Containers
-sudo -E make kernel-src
-sudo -E make kernel
+$ # Pull and setup latest kernel for Clear Containers
+$ sudo -E make kernel-src
+$ sudo -E make kernel
 ```
 
-## Use new generated kernel and image with cc-oci-runtime ##
+## Using the newly generated custom rootfs and kernel images
 
-To use the new generated kernel or image copy the defaults file:
+### Clear Containers 3.x
 
-```
-sudo mkdir -p /etc/cc-oci-runtime
-sudo cp /usr/share/defaults/cc-oci-runtime/vm.json /etc/cc-oci-runtime/vm.json
-```
+This section covers using the new resources with `cc-runtime`.
 
-And modify the paths for your new kernel and image:
+#### Installing the custom roots image
 
-```
-{
-	"vm": {
-		"path": "QEMU PATH...",
-		"image": "FULL IMAGE NAME ",
-		"kernel": {
-			"path": "FULL KERNEL NAME",
-			"parameters": "CMDLINE .."
-		}
-	}
-}
-```
+1. Install the image file
+   ```
+   $ sudo install --owner root --group root --mode 0755 workdir/container.img /usr/share/clear-containers/
+   ```
 
-## Requirements
+1. Update the runtime configuration for the image
+   ```
+   $ # (note that this is only an example using default paths).
+   $ sudo sed -i.bak -e 's!^\(image = ".*"\)!# \1 \
+   image = "/usr/share/clear-containers/container.img"!g' \
+   /usr/share/defaults/clear-containers/configuration.toml
+   ```
```

Comment:
We are copying  from the runtime docs, I think we can add a target to install the image and kernel and create a configuration under `/etc/clear-containers/configuration.toml` based on `/usr/share/defaults/clear-containers/configuration.toml`

Reply:
Good idea - I've captured that in #31.